src/main/java/gregtech/api/mui/GTGuiTextures.java

```diff
@@ -21,20 +23,32 @@ public class GTGuiTextures {
     /** Keys used for GT assets registered for use in Themes */
     public static class IDs {
 
-        public static final String STANDARD_BACKGROUND = "gregtech_standard_bg";
-        public static final String COVER_BACKGROUND = "gregtech_cover_bg";
-        public static final String BRONZE_BACKGROUND = "gregtech_bronze_bg";
-        public static final String STEEL_BACKGROUND = "gregtech_steel_bg";
-        public static final String PRIMITIVE_BACKGROUND = "gregtech_primitive_bg";
+        public static final String STANDARD_BACKGROUND = id("standard_bg");
+        public static final String COVER_BACKGROUND = id("cover_bg");
+        public static final String BRONZE_BACKGROUND = id("bronze_bg");
+        public static final String STEEL_BACKGROUND = id("steel_bg");
+        public static final String PRIMITIVE_BACKGROUND = id("primitive_bg");
+
+        public static final String STANDARD_SLOT = id("standard_slot");
+        public static final String BRONZE_SLOT = id("bronze_slot");
+        public static final String STEEL_SLOT = id("steel_slot");
+        public static final String PRIMITIVE_SLOT = id("primitive_slot");
 
-        public static final String STANDARD_SLOT = "gregtech_standard_slot";
-        public static final String BRONZE_SLOT = "gregtech_bronze_slot";
-        public static final String STEEL_SLOT = "gregtech_steel_slot";
-        public static final String PRIMITIVE_SLOT = "gregtech_primitive_slot";
+        public static final String STANDARD_FLUID_SLOT = id("standard_fluid_slot");
 
-        public static final String STANDARD_FLUID_SLOT = "gregtech_standard_fluid_slot";
+        public static final String STANDARD_BUTTON = id("standard_button");
+        public static final String BRONZE_BUTTON = id("bronze_button");
+        public static final String STEEL_BUTTON = id("steel_button");
+        public static final String BRONZE_BUTTON_SELECTED = id("bronze_button_sel");
+        public static final String STEEL_BUTTON_SELECTED = id("steel_button_sel");
 
-        public static final String STANDARD_BUTTON = "gregtech_standard_button";
+        public static final String DISPLAY = id("display");
+        public static final String DISPLAY_BRONZE = id("display_bronze");
+        public static final String DISPLAY_STEEL = id("display_steel");
+
+        private static String id(String path) {
```

Comment:
Doesn't this function already exist in GTUtilities or something? GTID?

Reply:
yes, but it returns a ResourceLocation